guides/v2.1/cloud/project/project-conf-files_magento-app.md

```diff
@@ -79,18 +79,27 @@ The `web` property defines how your application is exposed to the web (in HTTP).
--  `blacklist`: A list of files that should never be executed. Has no effect on static files.
--  `whitelist`: A list of static files (as regular expressions) that can be served. Dynamic files (for example, PHP files) are treated as static files and have their source code served, but they are not executed.
--  `expires`: The number of seconds whitelisted (that is, static) content should be cached by the browser. This enables the cache-control and expires headers for static content. The `expires` directive and resulting headers are left out entirely if this is not set.
+Attribute | Description
+--------- | -----------
+`document_root` | The path relative to the root of the application that is exposed on the web. Typical values include `/public` and `/web`.
+`passthru` | The URL used in the event a static file or PHP file could not be found. This would typically be your applications front controller, often `/index.php` or `/app.php`.
+`index_files` | To use a static file (for example, `index.html`) to serve your application. This key expects a collection. For this to work, the static file(s) should be included in your whitelist. For example, to use a file named `index.html` as an index file, your whitelist should include an element that matches the filename, like `- \.html$`.
+`blacklist` | A list of files that should never be executed. Has no effect on static files.
+`whitelist` | A list of static files (as regular expressions) that can be served. Dynamic files (for example, PHP files) are treated as static files and have their source code served, but they are not executed.
+`expires` | The number of seconds whitelisted (that is, static) content should be cached by the browser. This enables the cache-control and expires headers for static content. The `expires` directive and resulting headers are left out entirely if this is not set.
 
-Contrary to standard `.htaccess` approaches, which accept a *blacklist* and allow everything to be accessed except a specific list, we accept a *whitelist*, which means that anything not matched will trigger a 404 error and will be passed through to your `passthru` URL.
+Contrary to standard `.htaccess` approaches that accept a _blacklist_ and allow access to everything not on a specific list, we accept a _whitelist_, which means that anything not matched will trigger a 404 error and will be passed through to your `passthru` URL.
 
 Our default configuration allows the following:
 
--  From the root (`/`) path, only web, media, and `robots.txt` files can be accessed
+-  From the root (`/`) path, only web, media, and `robots.txt` files can be accessed  
+    For versions 2.1.4 to 2.1.10, the rules entry for the `robots.txt` file:
```

Comment:
Maybe it's just me, but the lead-in for this example sounds incomplete. I would expect to see a convention similar to line 105. For example:

> For versions 2.1.4 to 2.1.10, the following example shows the required rules entry for the `robots.txt` file:

Which begs the question, what about versions other than 2.1.4 to 2.1.10? Is there a different entry for other 2.1.x versions that I need to be aware of?

Reply:
Yeah, I can make that change.
It is the exception to the default, so maybe it can state that, too.